### PR TITLE
docs: update doc for guide user to config auth etcd

### LINKED ISSUE #8833: if etcd is use auth
### Description

if etcd is use auth,any affect?we use ten more cluster of one etcd,the etcd have 10000 connections,if auth is open,will affect service?and we use domain of etcd,there more error in parse domain in the error log.

### Environment

- APISIX version (run `apisix version`):2.15.1
- Operating system (run `uname -a`):
- OpenResty / Nginx version (run `openresty -V` or `nginx -V`):
- etcd version, if relevant (run `curl http://127.0.0.1:9090/v1/server_info`):
- APISIX Dashboard version, if relevant:
- Plugin runner version, for issues related to plugin runners:
- LuaRocks version, for installation issues (run `luarocks --version`):

### PR DESCRIPTION
**Pull request by mscb402**
### Description

This section of the document "How to configure ETCD auth for APISIX" is indeed missing. 

Fixes #8833

### Checklist

- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Comment by mscb402** [comment]
@spacewander @tokers  please help me review

File: docs/en/latest/FAQ.md [hunk 1]
```diff
@@ -704,6 +704,21 @@ Another solution is to switch to an experimental gRPC-based configuration synchr
     prefix: "/apisix"
 ```
 
+## How does APISIX configure ETCD with authentication
+
+If you are using an auth ETCD instance, you need to use the correct user and password to access it. You need to do edit this in the `conf/config.yaml` file.
```
Comment (tokers) [inline docs/en/latest/FAQ.md:709]:
```suggestion
Suppose you have an ETCD cluster that enables the auth. To access this cluster, you need to configure the correct username and password for Apache APISIX. Edit the `conf/config.yaml` file as follows:
```

File: docs/zh/latest/FAQ.md [hunk 2]
```diff
@@ -707,6 +707,21 @@ make GOOS=linux GOARCH=amd64
     prefix: "/apisix"
 ```
 
+## APISIX 如何配置带认证的 ETCD
+
+如果你使用的 ETCD 实例，是需要使用正确的用户和密码才能访问。你需要在`conf/config.yaml`文件中这样配置。
```
Comment (tokers) [inline docs/zh/latest/FAQ.md:712]: Update the sentence as per the English ones.

Comment (mscb402) [reply docs/zh/latest/FAQ.md]: done

File: docs/zh/latest/FAQ.md [hunk 3]
```diff
@@ -707,6 +707,21 @@ make GOOS=linux GOARCH=amd64
...
+```yaml
+deployment:
+  etcd:
+    host:
+      - "http://127.0.0.1:2379"
+    user: root                     # root username for etcd
+    password: 5tHkHhYkjr6cQY       # root password for etcd
+```
+
+关于 ETCD 的其他配置，比如过期时间、重试次数等等，你可以查看`conf/config-default.yaml`文件中的`ETCD`部分。
```
Comment (leslie-tsang) [inline docs/zh/latest/FAQ.md:723]:
```suggestion
关于 ETCD 的其他配置，比如过期时间、重试次数等等，你可以参考 `conf/config-default.yaml` 文件中的 `ETCD` 部分。
```

File: docs/zh/latest/FAQ.md [hunk 4]
```diff
@@ -707,6 +707,21 @@ make GOOS=linux GOARCH=amd64
     prefix: "/apisix"
 ```
 
+## APISIX 如何配置带认证的 ETCD
+
+假设您有一个启用身份验证的 ETCD 集群。要访问该集群，需要为 Apache APISIX 配置正确的用户名和密码。需要配置`conf/config.yaml` 文件，添加如下内容：
```
Comment (leslie-tsang) [inline docs/zh/latest/FAQ.md:712]:
```suggestion
假设您有一个启用身份验证的 ETCD 集群。要访问该集群，需要为 Apache APISIX 配置正确的用户名和密码。需要配置 `conf/config.yaml` 文件，添加如下内容：
```

File: docs/en/latest/FAQ.md [hunk 5]
```diff
@@ -704,6 +704,21 @@ Another solution is to switch to an experimental gRPC-based configuration synchr
...
+
+Suppose you have an ETCD cluster that enables the auth. To access this cluster, you need to configure the correct username and password for Apache APISIX. Edit the `conf/config.yaml` file as follows:
+
+```yaml
+deployment:
+  etcd:
+    host:
+      - "http://127.0.0.1:2379"
+    user: root                     # root username for etcd
+    password: 5tHkHhYkjr6cQY       # root password for etcd
```
Comment (leslie-tsang) [inline docs/en/latest/FAQ.md:717]:
```suggestion
    user: etcd_user                       # root username for etcd
    password: etcd_password     # root password for etcd
```
Would be better to don‘t have a specific password, Misunderstandings may arise.

File: docs/en/latest/FAQ.md [hunk 6]
```diff
@@ -704,6 +704,21 @@ Another solution is to switch to an experimental gRPC-based configuration synchr
...
+```yaml
+deployment:
+  etcd:
+    host:
+      - "http://127.0.0.1:2379"
+    user: etcd_user                       # root username for etcd
+    password: etcd_password     # root password for etcd
+```
+
+For other ETCD configurations, such as expiration times, retries, and so on, you can see the `ETCD` section in the `conf/config-default.yaml` file.
```
Comment (leslie-tsang) [inline docs/en/latest/FAQ.md:720]:
```suggestion
For other ETCD configurations, such as expiration times, retries, and so on, you can refer to the `ETCD` section in the `conf/config-default.yaml` file.
```

File: docs/en/latest/FAQ.md [hunk 7]
```diff
@@ -704,6 +704,21 @@ Another solution is to switch to an experimental gRPC-based configuration synchr
...
+
+Suppose you have an ETCD cluster that enables the auth. To access this cluster, you need to configure the correct username and password for Apache APISIX. Edit the `conf/config.yaml` file as follows:
+
+```yaml
+deployment:
+  etcd:
+    host:
+      - "http://127.0.0.1:2379"
+    user: etcd_user                       # root username for etcd
+    password: etcd_password     # root password for etcd
```
Comment (leslie-tsang) [inline docs/en/latest/FAQ.md:717]:
```suggestion
    user: etcd_user             # username for etcd
    password: etcd_password     # password for etcd
```
It is __not recommended__ to use `root` as etcd user.

File: docs/zh/latest/FAQ.md [hunk 8]
```diff
@@ -707,6 +707,21 @@ make GOOS=linux GOARCH=amd64
...
+
+假设您有一个启用身份验证的 ETCD 集群。要访问该集群，需要为 Apache APISIX 配置正确的用户名和密码。需要配置 `conf/config.yaml` 文件，添加如下内容：
+
+```yaml
+deployment:
+  etcd:
+    host:
+      - "http://127.0.0.1:2379"
+    user: root                     # root username for etcd
+    password: 5tHkHhYkjr6cQY       # root password for etcd
```
Comment (leslie-tsang) [inline docs/zh/latest/FAQ.md:720]:
```suggestion
    user: etcd_user             # username for etcd
    password: etcd_password     # password for etcd
```

File: docs/en/latest/FAQ.md [hunk 9]
```diff
@@ -704,6 +704,21 @@ Another solution is to switch to an experimental gRPC-based configuration synchr
     prefix: "/apisix"
 ```
 
+## How does APISIX configure ETCD with authentication
+
+Suppose you have an ETCD cluster that enables the auth. To access this cluster, you need to configure the correct username and password for Apache APISIX. Edit the `conf/config.yaml` file as follows:
```
Comment (tokers) [inline docs/en/latest/FAQ.md:709]:
```suggestion
Suppose you have an ETCD cluster that enables the auth. To access this cluster, you need to configure the correct username and password for Apache APISIX in `conf/config.yaml`:
```

File: docs/zh/latest/FAQ.md [hunk 10]
```diff
@@ -707,6 +707,21 @@ make GOOS=linux GOARCH=amd64
     prefix: "/apisix"
 ```
 
+## APISIX 如何配置带认证的 ETCD
+
+假设您有一个启用身份验证的 ETCD 集群。要访问该集群，需要为 Apache APISIX 配置正确的用户名和密码。需要配置 `conf/config.yaml` 文件，添加如下内容：
```
Comment (tokers) [inline docs/zh/latest/FAQ.md:712]:
```suggestion
假设您有一个启用身份验证的 ETCD 集群。要访问该集群，需要在 `conf/config.yaml` 中为 Apache APISIX 配置正确的用户名和密码：
```

File: docs/en/latest/FAQ.md [hunk 11]
```diff
@@ -704,6 +704,21 @@ Another solution is to switch to an experimental gRPC-based configuration synchr
     prefix: "/apisix"
 ```
 
+## How does APISIX configure ETCD with authentication
```
Comment (shreemaan-abhishek) [inline docs/en/latest/FAQ.md:707]:
```suggestion
## How does APISIX configure ETCD with authentication?
```

File: docs/zh/latest/FAQ.md [hunk 12]
```diff
@@ -707,6 +707,21 @@ make GOOS=linux GOARCH=amd64
     prefix: "/apisix"
 ```
 
+## APISIX 如何配置带认证的 ETCD
```
Comment (shreemaan-abhishek) [inline docs/zh/latest/FAQ.md:710]: make similar punctuation symbol changes here accordingly.

**Review by shreemaan-abhishek — CHANGES_REQUESTED** [review]

**Review by leslie-tsang — APPROVED** [review]
LGTM

**Review by tokers — APPROVED** [review]

**Review by nic-chen — APPROVED** [review]

**Pull request merged**